components/core/src/clp_s/CommandLineArguments.cpp

```diff
@@ -286,6 +286,20 @@ CommandLineArguments::parse_arguments(int argc, char const** argv) {
             );
             extraction_options.add(input_options);
 
+            po::options_description metadata_options("Metadata Options");
```

Comment:
Can we move this block after the decompression options block (following the order that we add them to `visible_options`)?

Reply:
Can we also rename it to `output_metadata_options` (and `Output Metadata Options`)? Currently, it's ambiguous whether this is an arg for existing input metadata or output metadata.

Reply:
Sure thing